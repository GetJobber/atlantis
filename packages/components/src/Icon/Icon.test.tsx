import React from "react";
import renderer from "react-test-renderer";
import { Icon } from ".";

it("renders dashboard icon", () => {
  const tree = renderer.create(<Icon iconName="dashboard" />).toJSON();
  expect(tree).toMatchSnapshot();
});

it("renders apple icon", () => {
  const tree = renderer.create(<Icon iconName="apple" />).toJSON();
  expect(tree).toMatchSnapshot();
});
