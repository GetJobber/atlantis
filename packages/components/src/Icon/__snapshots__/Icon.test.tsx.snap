// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders apple icon 1`] = `
<div
  className="apple icon"
/>
`;

exports[`renders dashboard icon 1`] = `
<div
  className="dashboard icon"
/>
`;
